### PR TITLE
Add notice to RAT-Partner page

### PR DESCRIPTION
**Pull request by larswmh**
image
chrome_WRNnmPgh6U

### CODE REVIEW

File: src/data/rat-partner.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {
+    "section-notice": {
+        "textblock": [
+            "Dear interested persons,",
+            "thank you for your interest in connecting to the Corona-Warn-App. You can find current information about this on <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki' target='_blank' rel='noopener noreferrer'>GitHub</a>.",
```
Comment (MikeMcC399) [inline src/data/rat-partner.json:5]: "thank you" => "Thank you"

In English correspondence format the first letter of the first word after the salutation is capitalized, unlike in German.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Minor format suggestion. See in-line comment.

File: src/data/rat-partner.json (hunk 1 above)
Comment (larswmh) [reply src/data/rat-partner.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/2889/commits/36582ff34e199fc79cddec94613d0c60a30cdd5a, thanks!

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for review.

**Pull request merged**